client/lib/oauth-token/index.js

```diff
@@ -29,13 +29,13 @@ export function getToken() {
 
 export function setToken( token ) {
 	// TODO: Support secure cookies if this is ever used outside of the desktop app
-	document.cookie = cookie.serialize( TOKEN_NAME, token, { maxAge: MAX_AGE } );
+	document.cookie = cookie.serialize( TOKEN_NAME, token, { maxAge: MAX_AGE, path: '/' } );
```

Comment:
We want to make sure we are setting OAuth cookie on the root path, and not on something like `/scan` or `/backup`.